cicd/3-app/javabuilder/config/test.config.json

```diff
@@ -2,10 +2,10 @@
   "Parameters": {
     "BaseDomainName": "code.org",
     "BaseDomainNameHostedZonedID": "Z2LCOI49SCXUGU",
-    "ProvisionedConcurrentExecutions": "1",
-    "ReservedConcurrentExecutions": "3",
-    "LimitPerHour": "15",
-    "LimitPerDay": "40",
+    "ProvisionedConcurrentExecutions": "5",
+    "ReservedConcurrentExecutions": "25",
+    "LimitPerHour": "50",
+    "LimitPerDay": "150",
```

Comment:
we should document somewhere how to reset yourself if you hit the limit

Reply:
Good call. It should probably belong in the same doc that DOTDs can reference for Javabuilder issues I think? I'll go ahead and start one and we can continue adding to it.

Reply:
Started a doc here: https://docs.google.com/document/d/1bHvV6pvUcwxgZpw0YWBmxFggQL5KqYx9zwolwkZhjU8/edit#